### PR TITLE
Docs: add a note about loading source with AMD

### PR DESCRIPTION
**Pull request by timmywil**
- Moves the note about the watch task and the note about loading with AMD to their own section under "Test Suite Tips"

Fixes gh-2714

### CODE REVIEW

**Comment by dmethvin** [comment]
All these changes LGTM!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -101,10 +101,10 @@ Run the build script
 $ npm run build
 ```
 
-Run the Grunt tools:
+Build a local copy of jQuery using grunt
 
 ```bash
-$ grunt && grunt watch
+$ grunt
```
Comment (markelog) [inline CONTRIBUTING.md]: Do we need that command at all?

Comment (timmywil) [inline CONTRIBUTING.md]: You do when following those steps to view the test page.

Comment (markelog) [inline CONTRIBUTING.md]: "npm run build" should take care of that, right now, it looks like you need to run both - grunt and npm commands

Regards,
Oleg

Comment (timmywil) [inline CONTRIBUTING.md]: Ah, of course. Didn't notice the `run build` line.

**Pull request closed without merging**